bookshelf.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+class Book:
+    def __init__(self, title, author, pages, price, quantity, sales):
+        self.title = title
+        self.author = author
+        self.pages = pages
+        self.price = price
+        self.quantity = quantity
+        self.sales = sales
```

Comment:
attributes should be private or protected

Reply:
Made them private